CHANGELOG.md

```diff
@@ -6,9 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Removed
+- Method `setCardFormData` and `cardFormData` from payment context.
```

Comment:
```suggestion
- Methods `setCardFormData` and `cardFormData` from payment context.
```

Reply:
the method is only the `setCardFormData`, the other one was an object

Reply:
I have to agree this asymmetry makes the sentence a bit confusing. Might as well just remove `Method`.